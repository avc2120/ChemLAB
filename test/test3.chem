function hey(String x)
{
	print "hello";
}

function main()
{
	String y;
	y = "hello";
	Call hey(y);
}