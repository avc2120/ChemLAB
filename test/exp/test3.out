0
1
2
3
4
5
6
42
42
-42
-42
4.2
0.4242424242424242